tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,40 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Switcher to Masterhead
+    [Tags]    ODS-771
+    Go To RHODS Dashboard
+    Click Button    //button[@class="pf-c-app-launcher__toggle"]
+    ${cluster_id} =    Get Cluster ID
+    ${list_of_links} =    Get Links From Switcher
+    ${status}    Run Keyword And Return Status    Is Environment Staging    ${list_of_links}[1]
+    Check Links For OpenShift Clusetr Manager    ${list_of_links}[1]    ${cluster_id}    ${status}
+
 *** Keywords ***
+Is Environment Staging
+    [Arguments]    ${link}
+    Should Contain    ${link}    qaprodauth
+
+Check Links For OpenShift Clusetr Manager
+    [Arguments]    ${link}    ${cluster_id}    ${is_env_type_stage}
+    ${cluster_id} =    Remove String    ${cluster_id}    "
+    IF    "${is_env_type_stage}" == "True"
+        Should Be Equal    ${link}    https://qaprodauth.cloud.redhat.com/openshift/details/${cluster_id}
+    ELSE
+        Should Be Equal    ${link}    https://cloud.redhat.com/openshift/details/${cluster_id}
+    END
```

Comment:
could you also add a HTTP status check? Instead of checking it in `Get Links From Switcher`